orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -2092,6 +2092,40 @@ TEST(GradientUtilsTest, InPlaceAccumulatorFloat32) {
   test.Run();
 }
 
+TEST(GradientUtilsTest, InPlaceAccumulatorV2_CUDA) {
```

Comment:
I think you need put your CUDA tests within "#if defined(USE_CUDA) || defined(USE_ROCM)"